config/config.go

```diff
@@ -792,6 +792,7 @@ type KubernetesSDConfig struct {
 	RetryInterval   model.Duration `yaml:"retry_interval,omitempty"`
 	RequestTimeout  model.Duration `yaml:"request_timeout,omitempty"`
 	TLSConfig       TLSConfig      `yaml:"tls_config,omitempty"`
+	Namespace 	string 	       `yaml:"namespace,omitempty"`
```

Comment:
Namespace already has a significant meaning in k8, is there a less ambiguous term for this?